freesound/local_settings.example.py

```diff
@@ -104,3 +105,5 @@
 RAVEN_CONFIG = {
     'dsn': 'DSN_PROVIDED_BY_SENTRY',
 }
+
+AWS_SQS_QUEUE_URL = ''  # also proper AWS credentials should be configured in ~/.aws
```

Comment:
Make sure that you also open a pull request on the freesound-deploy repository to include the actual queue url.

Is `~/.aws` the only way of doing credentials? I'm not sure how well this will work for us in production, it would be great if there was an alternative way of setting the credentials (via environment variables?)

Reply:
No, it is possible to specify credentials in code. I will update the config to reflect that, I was not sure what would be the preferred way of doing this.